src/net/sourceforge/kolmafia/textui/parsetree/Value.java

```diff
@@ -25,6 +25,8 @@
  * as some sort of... hazmat suit..?
  */
 public class Value implements TypedNode, Comparable<Value> {
+  public static final Value BAD_VALUE = new Value(new Type.BadType(null, null));
```

Comment:
Do we want BAD_VALUE.isBad() to be true? That doesn't seem to be the case.

Reply:
BAD_VALUE.isBad() doesn't exist.
isBad() is for Symbol
to check foe BAD_VALUE, we use Evaluable.evaluatesTo(BAD_VALUE)